testing/src/cypress/e2e/02-institution/03-institution-profile.cy.ts

```diff
@@ -2,71 +2,165 @@ import DashboardInstitutionObject from "../../page-objects/Institution-objects/D
-    cy.intercept("GET", "**/institution").as("institution");
-    dashboardInstitutionObject.manageInstitutionButton().click();
-    dashboardInstitutionObject.allLocationsText().should("be.visible");
-    institutionObject.manageProfileButton().click();
-    cy.wait("@institution");
-    institutionObject.primaryPhoneNumberInputText().should("be.disabled");
-    institutionObject.primaryEmailInputText().should("be.disabled");
-    institutionObject.establishedDate().should("be.disabled");
-    institutionObject.firstNameInstitutionInputText().should("be.disabled");
-    institutionObject.lastNameInstitutionInputText().should("be.disabled");
-    institutionObject.emailInstitutionInputText().should("be.disabled");
-    institutionObject.phoneNumberInstitutionInputText().should("be.disabled");
-
     institutionObject
       .addressInstitutionInputText()
       .clear()
       .type(profileData.addressInput);
     institutionObject.cityInputText().clear().type(profileData.cityInput);
     institutionObject.countryInputText().click();
-    institutionObject
-      .countrySearchInputText(profileData.countryInput)
-      .type("{enter}");
+    institutionObject.countrySearchInputText(profileData.countryInput);
+
     institutionObject.provinceInputText().click({ force: true });
     institutionObject
       .provinceSearchInputText(profileData.provinceState)
       .type("{enter}");
     institutionObject.postalInputText().clear().type(profileData.postalInput);
   });
+
+  it("Verify that address line 1 should be a mandatory field", () => {
+    institutionObject.verifyThatFieldShouldNotBeEmpty(
+      institutionObject.addressInstitutionInputText(),
+      "Address line 1"
+    );
+  });
+
+  it("Verify that address line 1 should not be more than 100 chars", () => {
+    institutionObject.verifyThatFieldDoesNotAcceptMoreThanSpecificChars(
+      institutionObject.cityInputText(),
+      "City"
+    );
+  });
+
+  it("Verify that city should be a mandatory field", () => {
+    institutionObject.verifyThatFieldShouldNotBeEmpty(
+      institutionObject.cityInputText(),
+      "City"
+    );
+  });
+
+  it("Verify that city should not be more than 100 chars", () => {
+    institutionObject.verifyThatFieldDoesNotAcceptMoreThanSpecificChars(
+      institutionObject.addressInstitutionInputText(),
+      "City"
+    );
+  });
+
+  it("Verify that country field is a drop down and contains two values", () => {
+    const countryOptions = ["Canada", "other"];
+    institutionObject.countryInputText().click();
+    countryOptions.forEach((country) => {
+      institutionObject.verifyTheDropDownOptionsAreVisible(
+        institutionObject.countryDropdownMenu(),
+        country
+      );
+    });
+  });
+
+  it("Verify that selecting Canada from country dropdown makes the pre-populate the provinces available", () => {
+    institutionObject.countryInputText().click({ force: true });
```

Comment:
just curios, what does `force: true`?

Reply:
It does forces the action without any errors like if there is any other element floating on the subject or even an element is disabled. 

But in this context - it clicks on the dropdown menu even if the dropdown is already expanded.